packages/base-plugin/lib/plugin.js

```diff
@@ -1,68 +1,53 @@
 import {logger} from '@appium/support';
 
-export default class BasePlugin {
-  // plugins can define new methods for the Appium server to map to command names, of the same
```

Comment:
i don't see this helpful comment reproduced in the `Plugin` interface

Reply:
ah, whoops.  I've added it to `PluginStatic`.